### DISCUSSION TITLE
Eloquent Events

### DISCUSSION THREAD

**Discussion by dominikgeimer** [ryangjchandler/orbit#131] [Q&A] [answered]
Hello,

there is no way to trigger an eloquent event, when a new file is created in the content folder, correct?

Thanks!
Dominik

**Comment by ryangjchandler** [comment, marked as answer]
@DominikGeimer Correct. If you manually create a file in the content folder, that's not related to Eloquent at all. The file will be picked up by Orbit the next time the model is booted though.

**Reply by dominikgeimer** [reply]
Thanks for your reply.
